preprocessing/berlin/raw/markets-berlin.json

```diff
@@ -718,10 +718,10 @@
                     "www": "",
                     "zeiten": "10:00 - 19:00\n09:00 - 16:00"
                 },
-                "description": "Mo - Fr\nSa<br />10:00 - 19:00\n09:00 - 16:00<br /> <a href=\"http://www.berlin.de/sen/wirtschaft/service/maerkte-feste/wochen-troedelmaerkte/index.php/detail/45\">Mehr...</a>",
+                "description": "<strong>Wochenmarkt Am Kaufhof Ostbahnhof\n(Erich-Steinfurth-Straße)</strong><br />Mo - Fr\nSa<br />10:00 - 19:00\n09:00 - 16:00<br /> <a href=\"http://www.berlin.de/sen/wirtschaft/service/maerkte-feste/wochen-troedelmaerkte/index.php/detail/45\">Mehr...</a><br /><br /><strong>design.Börse_Berlin in Galeria Kaufhof am Ostbahnhof, Erich-Steinfurth-Straße</strong><br />22. - 24.01.2016<br />Fr 16:00-20:00\nSa 12:00-20:00\nSo 13:00-18:00<br /> <a href=\"http://www.berlin.de/sen/wirtschaft/service/maerkte-feste/wochen-troedelmaerkte/index.php/detail/61\">Mehr...</a>",
                 "href": "http://www.berlin.de/sen/wirtschaft/service/maerkte-feste/wochen-troedelmaerkte/index.php/detail/45",
                 "id": "http://www.berlin.de/sen/wirtschaft/service/maerkte-feste/wochen-troedelmaerkte/index.php/detail/45",
-                "title": "Wochenmarkt Am Kaufhof Ostbahnhof\n(Erich-Steinfurth-Straße)"
+                "title": "Mehrerer Einträge..."
```

Comment:
@johnjohndoe didn't you make sure in the last commit that we do not have `Mehrere Einträge` titles anymore?